notebooks/exploring_explorer.livemd

```diff
@@ -4,19 +4,17 @@
 
 Connect to a `Mix` project with `Explorer` installed or:
 
-<!-- livebook:{"force_markdown":true} -->
```

Comment:
I couldn't understand the intention here. I removed in order to run the Elixir cell. Is that OK?

Reply:
Definitely okay. I'm not sure why I had set it to markdown in the first place.